cytotable/convert.py

```diff
@@ -43,10 +43,7 @@ def _get_table_columns_and_types(
             list of dictionaries which each include column level information
     """
 
-    import pathlib
-
```

Comment:
Why is pathlib removed?

Reply:
Thanks - `pathlib` was removed as I noticed it was no longer needed for `_get_table_columns_and_types`. I believe the import inadvertently remained after earlier changes than this PR, so this was a bit of opportunistic refactoring (I noticed it during the work towards this PR).